components/mdi/Core/Inc/can.h

```diff
@@ -46,6 +46,7 @@ typedef struct
 	bool PWR_mode_on;
 	bool regen;
 	bool cruise_control_enable;
+	uint32_t last_rx_time;
```

Comment:
We talked about moving this to a global variable of its own; this introduces a lot of changes that need to be made in the MDI firmware (after comp). This is good for now.